lib/valkyrie/specs/shared_specs/indexer.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+RSpec.shared_examples 'a Valkyrie::Indexer' do |*_flags|
```

Comment:
`Valkyrie::Persistence::Solr::Indexer`? In case we ever have an ElasticSearch adapter.